facilitator/src/bin/facilitator.rs

```diff
@@ -709,17 +729,43 @@ fn intake_batch(sub_matches: &ArgMatches) -> Result<(), anyhow::Error> {
         batch_signing_key: batch_signing_key_from_arg(sub_matches)?,
     };
 
+    let batch_id: &str = sub_matches.value_of("batch-id").unwrap();
+    let batch_id: Uuid = Uuid::parse_str(batch_id).unwrap();
+
+    let date: &str = sub_matches.value_of("date").unwrap();
+    let date: NaiveDateTime = NaiveDateTime::parse_from_str(date, DATE_FORMAT).unwrap();
+
     let mut batch_intaker = BatchIntaker::new(
         &sub_matches.value_of("aggregation-id").unwrap(),
-        &Uuid::parse_str(sub_matches.value_of("batch-id").unwrap()).unwrap(),
-        &NaiveDateTime::parse_from_str(&sub_matches.value_of("date").unwrap(), DATE_FORMAT)
-            .unwrap(),
+        &batch_id,
+        &date,
         &mut intake_transport,
         &mut peer_validation_transport,
         &mut own_validation_transport,
         is_first_from_arg(sub_matches),
     )?;
-    batch_intaker.generate_validation_share()
+
+    let intake_started: Counter = register_counter!(
+        "intake_jobs_begun",
+        "Number of intake-batch jobs that started (on the facilitator side)"
```

Comment:
nit: I think we should remove the `(on the facilitator side)` suffix from this and other counters. I think whether or not this was a facilitator or a PHA can be inferred from whether the whole env `is_first` or not.

Reply:
In this case I am distinguishing the variable from `intake_jobs_started` on the workflow-manager side. In other words, we have one metric for "how many jobs did workflow-manager create," and "how many jobs did the facilitator actually start working on." They'll almost always be the same, but it seemed like a very simple first metric to add.

Reply:
Oh, I understand now. I was thinking about "facilitator" as in "not a PHA", but you meant "facilitator" as in "not workflow-manager". This is fine as is.